projects/nginx/auth_fuzz/inputs/in1

```diff
@@ -0,0 +1 @@
+HEAD / HTTP/1.1
```

Comment:
Maybe it is better to zip all of these inputs and unzip them in build.sh?

Reply:
Done